app/src/main/java/org/simple/clinic/home/overdue/OverdueScreenController.kt

```diff
@@ -51,6 +53,11 @@ class OverdueScreenController @Inject constructor(
     return updateListStream.mergeWith(emptyStateStream)
   }
 
+  private fun getDaysAgoForInstant(instant: Instant): Int {
```

Comment:
I think we should avoid using `get` prefix for functions here

Reply:
Yeah this should definitely be changed.